### DISCUSSION TITLE
Moving around components in Budibase

### DISCUSSION THREAD

**Discussion by slendertech** [Budibase/budibase#7216] [Help]
How the heck do I position components in Budibase? I can't click and drag, and the whole margin settings on the right panel isn't customizable enough. If I can, I would like to completely turn off the auto-alignment thing and just be able to drag the component around the screen.

**Comment by JayP718** [comment]
Hi @slenderstudios,

From my understanding that's not possible currently... Budibase uses containers for positioning.

It's a little different from typical drag and drop interfaces but it's great once I got the handle of it and often lead me to create screens with a better UI. 

Do you have a sketch/specific example of what you are trying to do, perhaps I can help with some pointers?
